yaml/landuse.yaml

```diff
@@ -1116,3 +1123,23 @@ filters:
       <<: *output_properties
       kind: crane
       kind_detail: {col: "crane:type"}
+  - filter:
+      any:
+        harbour: 'yes'
+        landuse: harbour
+    min_zoom: 16
+    output:
+      <<: *output_properties
+      kind: harbor
```

Comment:
Our practice on the kinds is generally to follow British English, especially if that's what the original OSM tag was. 

Please set this to `harbour`.

Reply:
See PR comments – this should move to `landuse: harbour` only.

Reply:
`harbor` -> `harbour` in 96110df.